posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc

```diff
@@ -8,8 +8,8 @@ author_github: https://github.com/kevin-ortega
 seo-title:  Deploy Java applications to IBM Cloud Code Engine (ICCE) using the Paketo Buildpack for Liberty - OpenLiberty.io
 seo-description: Learn to deploy your applications to ICCE easily with the Paketo Buildpack for Liberty
 blog_description: "Learn to deploy your applications to ICCE easily with the Paketo Buildpack for Liberty"
-open-graph-image: https://openliberty.io/img/blog/code_engine_build_step_details.png
-open-graph-image-alt: screen capture of Code Engine build details
+open-graph-image: /img/blog/code_engine_build_strategy.png
```

Comment:
you'll need to check to see if this works...... from my recollection relative links for og use to not work.

Reply:
ok- I'll test it as KIn suggested in the other PR

Reply:
I doubled checked this suggestion since it was mine in the previous PR.
It seems like the safest approach is to use absolute paths. 

There are inconsistencies on whether certain social media platforms support relative paths in `og:image`.  Since the examples on [Open Graph](https://ogp.me/) are all absolute paths, implementers may have taken absolute paths as a requirement.

My suggestion is to just keep `open-graph-image: https://openliberty.io/img/blog/code_engine_build_step_details.png` and visually check that the path looks correct with the merge of the actual image used in `og:image`.